types/types.go

```diff
@@ -202,7 +204,8 @@ func GetInstanceManagerLabels(node, instanceManagerImage string, managerType Ins
 		labels[GetLonghornLabelKey(LonghornLabelNode)] = node
 	}
 	if instanceManagerImage != "" {
-		labels[GetLonghornLabelKey(LonghornLabelInstanceManagerImage)] = GetImageCanonicalName(instanceManagerImage)
+		imageCanonicalName := GetImageCanonicalName(instanceManagerImage)
```

Comment:
Don't need this variable since it's straightforward to call it inside `GetInstanceManagerImageChecksumName`

Reply:
Will fix it. Thanks